test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Follow-up: The migration added to this PR will add `https://` to the beginning of all protocol-less values of `domain` and `demoDomain`. It will change any that start with `http://` to `https://`